packages/scss/src/components/_page-header.scss

```diff
@@ -0,0 +1,45 @@
+.pageHeader {
+	@include elevate(1);
+	padding: _theme('spacings.standard') 2.5rem;
+	background-color: _color('white');
+
+	&.mod-withMenu {
+		padding-bottom: 0;
+	}
+
+	.breadcrumbs {
+		margin-bottom: 0;
+	}
+
+	.menu {
+		&::after {
+			display: none;
+		}
+	}
+}
+
+.pageHeader-content {
+	display: flex;
+	justify-content: space-between;
+	align-items: flex-start;
+	flex-wrap: wrap;
+}
+
+.pageHeader-content-title,
+.pageHeader-content-actions {
+	display: flex;
+	align-items: center;
+	flex-wrap: wrap;
+}
+
+.pageHeader-content-actions {
+	& > *:not(:last-child){
+		margin-right: _theme('spacings.small');
+	}
+}
+
+.pageHeader-description {
+	max-width: 50%;
```

Comment:
A flat 50% will be problematic in a responsive context. We should keep the max-width: 50rem but make it themable